packages/react-native-aztec/android/gradlew.bat

```diff
@@ -14,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"

```

Comment:
Out of curiosity, did you set these or where out of the box changes from migrating to Gradle version 6.1.1? Same for the ones in `gradlew`.

Reply:
No, these are all generated by the `./gradlew wrapper --gradle-version 6.1.1 --distribution-type all` command. You can find more info about the Gradle upgrade in https://github.com/WordPress/gutenberg/pull/28305